packages/authentication/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/authentication/FirebaseAuthentication.java

```diff
@@ -1,5 +1,7 @@
 package io.capawesome.capacitorjs.plugins.firebase.authentication;
 
+import static io.capawesome.capacitorjs.plugins.firebase.authentication.FirebaseAuthenticationHelper.*;
```

Comment:
Please do not use `import static` for the `FirebaseAuthenticationHelper` (all files).